plugins/telemetry-plugin/src/telemetry-plugin.ts

```diff
@@ -21,6 +21,51 @@ export function start(context: theia.PluginContext) {
                 ['programming language', e.document.languageId]
             ]);
     });
+
+    let gitLogHandlerInitialized: boolean;
+    /* Git log handler, listens to Git commands and pushes telemetry events. */
+    const onChange = () => {
+        // Get the vscode Git plugin if the plugin is started.
+        const gitExtension = theia.plugins.getPlugin('vscode.git');
+        if (!gitLogHandlerInitialized && gitExtension && gitExtension.exports) {
+            // Set the initialized flag to true state, to not to initialize the handler again on plugin change event.
+            gitLogHandlerInitialized = true;
+            // tslint:disable-next-line:no-any
+            const git: any = gitExtension.exports._model.git;
+            let command: string;
+            let url: string;
+            const listener = async (out: string) => {
+                // Parse Git log events.
+                const split = out.split(' ');
+                if (out.startsWith('> git commit') || out.startsWith('> git push')) {
+                    command = split[2];
+                    url = split[3];
+                    switch (command) {
+                        case 'commit': {
+                            che.telemetry.event('GIT_USED', context.extensionPath,
```

Comment:
According to [this code in the existing telemetry backend base](https://github.com/che-incubator/che-workspace-telemetry-client/blob/master/backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/base/AnalyticsEvent.java#L24), the name of the event would be: `COMMIT_LOCALLY`

Additionally, for now we don't have any parameter related to the commit event (no URL).

It could be added in the future (if PM agrees), but then it should be added in the [`AnalyticsEvents`](https://github.com/che-incubator/che-workspace-telemetry-client/blob/master/backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/base/AnalyticsEvent.java) and [`EventProperties`](https://github.com/che-incubator/che-workspace-telemetry-client/blob/master/backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/base/EventProperties.java) files

Reply:
done